Pod/Classes/public/error/Error.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  Error.swift
+//  Pods
+//
+//  Created by Jake Young on 6/14/16.
+//
+
+import Foundation
+
+/// Debugging levels for WistiaKit
+public enum DebuggingLevel {
+    
+    /// Shhh, fust be silent.
+    case None
+    
+    /// Print the high level activity and errors.
+    case Friendly
+    
+    /// Print all the things.
+    case Annoying
+}
+
+internal func printErrorForDebuggingLevel(debuggingLevel debuggingLevel: DebuggingLevel, message: String? = nil, error: NSError? = nil) {
+    
+    switch debuggingLevel {
+        
+    case .None:
+        break
+    case .Friendly:
+        
+        // We print the message and fallthrough to allow .Annoying case to add to things.
+        guard let message = message else { break }
+        print(message)
+        fallthrough
```

Comment:
This `fallthrough` affects the `.Friendly` case.  So, if debugging is `.Friendly`, it falls though and executes the `.Annoying` case's code as well.  From a cursory read, it seems this wasn't the intention (ie. annoying should print everything, friendly should print less).